packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

```diff
@@ -4,218 +4,231 @@ title: OAuth 2.0 and OpenID Connect Overview
 
 # OAuth 2.0 and OpenID Connect Overview
 
-This article provides a high-level introduction to OAuth 2.0 and OpenID Connect (OIDC), which are the standard protocols that Okta's authentication and authorization solutions are based on. This article discusses how you can implement flows based on these standards using Okta, and what flows and grant types are commonly used by the different types of apps.
+OAuth 2.0 and OpenID Connect (OIDC) are industry standard protocols for user authentication and authorization. Okta identity solutions are based on these standards.
 
-> **Note**: See [Okta deployment models &mdash; redirect vs. embedded](/docs/concepts/redirect-vs-embedded/) for more information on the specific types of authentication deployment models that Okta provides that are built on top of OAuth 2.0 and OIDC.
+---
 
-## OAuth 2.0 vs OpenID Connect
+**Learning outcomes**
+
+* Learn how to implement flows based on OAuth 2.0 and OIDC using Okta.
+* Learn what flows and grant types are commonly used by different types of applications.
 
-There are two main types of authentication that you can perform with Okta:
+> **Note**: To learn about the Okta authentication deployment models built on top of OAuth 2.0 and OIDC, see [Okta deployment models](/docs/concepts/redirect-vs-embedded/).
 
-* The [OpenID Connect](/docs/reference/api/oidc/) (OIDC) protocol is built on the OAuth 2.0 protocol and helps authenticate users and convey information about them. It's also more opinionated than plain OAuth 2.0, for example in its scope definitions.
+## OAuth 2.0 vs OpenID Connect
 
-* The [OAuth 2.0](/docs/reference/api/oidc/) protocol controls authorization to access a protected resource, like your web app, native app, or API service.
+OAuth 2.0 and OpenID Connect (OIDC) are complementary protocols, defining how users are first authenticated by a server and then authorized to access resources:
```

Comment:
I wouldn't use "first" here because it isn't really a sequence of events. This makes it sound like the user is authenticated at Okta using OIDC, and then later uses OAuth to get authorized to access resources. In reality, authentication at Okta is using Okta's own internal mechanisms like passwords, email magic link, passkeys, etc. Users then use OIDC and/or OAuth to access applications.

Reply:
Fair enough. I'll simply remove "first".